.github/workflows/verify.yaml

```diff
@@ -22,6 +24,9 @@ jobs:
         run: git diff --exit-code
 
   Test:
+    permissions:
+      checks: write
+      pull-requests: write
```

Comment:
Could you explain me what this does and why this is needed?

Reply:
If this permission is not there, then we will get a 403. For example, here when these permissions were not added 
https://github.com/agera-edc/RegistrationService/runs/7803472074?check_suite_focus=true#step:9:49